### PR TITLE
Spoofing explained

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-40e9abfcb1/how-to/)

**Comment by stephamon** [comment]
Hi self, here are some edits:

[Email.spoofing.best.practices_SFedit.docx](https://github.com/rackerlabs/rackspace-how-to/files/1471731/Email.spoofing.best.practices_SFedit.docx)
[Email.spoofing.best.practices_SFedit.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1471732/Email.spoofing.best.practices_SFedit.pdf)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a5c486f612/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-21397d5bf3/how-to/). :bow:

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+type: article
+created_date: '2017-09-25'
+created_by: William Loy
+last_modified_date: '2017-11-14'
+last_modified_by: Stephanie Fillmon
+product: Rackspace Email
+product_url: rackspace-email
+---
+
+This article explains email spoofing, and describes the steps that you can take to combat it.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: You can remove the comma after spoofing.

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- **Time needed:** Approximately 15 minutes
+- **Tools required:**  Email access
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+If you would prefer a video tutorial see [Rackspace Email - Spoofing: How to identify & protect your organization](https://emailhelp.rackspace.com/l/how-to-prevent-email-spoofing).
+
+### What is email spoofing?
+
+The word **spoof** means **falsified**. A spoofed email is when the sender purposely alters parts of the email to make the message appear as though it was authored by someone else. Commonly, the sender’s name and email address and the body of the message are formatted to appear from a legitimate source. Sometimes, the spoofer makes the email appear to come from a private citizen somewhere.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: A spoofed email is one in which the sender....

Also delete somewhere (after private citizen).

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+A spoofed message can appear to come from a coworker, a bank, a family member or any number of seemingly trustworthy sources. A good spoof looks like any other email that you normally receive.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: family member, or

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+**Warning:** If you suspect you have received a fraudulent message, **DO NOT** click any link in the message or enter any information that is requested.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: If you suspect that you have...

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+### Why do people spoof email?
+
+In many cases, the spoofed email is part of a **phishing** (scam) attack. In other cases, a spoofed email is used to dishonestly market an online service or sell you a bogus product. The intent is to trick the recipient into making a damaging statement or releasing sensitive information, such as passwords. If you're receiving bounced (returned) emails for messages that you never sent, this could be a case of spoofing.
+
+### Identify a spoofed message
+
+Scammers alter different sections of an email to disguise who is the *actual* sender of the message. To view the following properties, you must open the **email headers** of a message that you suspect has been spoofed. Following are some examples of spoofed properties:
+
+- **FROM** boss@companyexample.com - This appears to come from a legitimate source on any spoofed message.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: on a spoofed message.

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+- **REPLY-TO** - This can also be spoofed, but a lazy scammer will leave the actual **REPLY-TO** address. If you see a different sending address here, the email might have been spoofed.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: lazy scammer might leave..

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+- **RETURN-PATH** - This can also be spoofed, but a lazy scammer will leave the actual **RETURN-PATH** address. If you see a different sending address here, the email might have been spoofed.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: lazy scammer might leave...

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+- **SOURCE IP** address or “X-ORIGIN” address - This is typically more difficult to alter but it is possible.
+
+These first three properties can be easily altered by using settings in your Microsoft Outlook, Gmail, Hotmail, or other email software. The fourth property above, IP address, can also be altered, but usually requires more sophisticated user knowledge to make a false IP address convincing.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: The first three...

The fourth property, IP address,.....

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+In the following example, it appears that the recipient has received a message from their office assistant, requesting money.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: can remove the comma after assistant

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+<img src="{% asset_path rackspace-email/email-spoofing-explained/from_assistant.png %}" />
+
+The **subject** line should alert you immediately. This user should contact their assistant through another form of communication to confirm that they did not send this message. Next, you will want to discover who actually sent the message by opening the message headers. It should look similar to the following:
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: Next, you want to...

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+
+<img src="{% asset_path rackspace-email/email-spoofing-explained/reply_to.png %}" />
+
+In this message header snippet, we see that the **From:** field shows the message being sent from **"Assistant"\<assistant@yourdomainexample.com\>**. However, we can also see that the **REPLY-TO:** field lists *spoofer@scam.com*. That is a clear cut example of a spoofed message.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: That is a clear example...

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+<img src="{% asset_path rackspace-email/email-spoofing-explained/from_assistant.png %}" />
+
+The **subject** line should alert you immediately. This user should contact their assistant through another form of communication to confirm that they did not send this message. Next, you will want to discover who actually sent the message by opening the message headers. It should look similar to the following:
+
+<img src="{% asset_path rackspace-email/email-spoofing-explained/reply_to.png %}" />
+
+In this message header snippet, we see that the **From:** field shows the message being sent from **"Assistant"\<assistant@yourdomainexample.com\>**. However, we can also see that the **REPLY-TO:** field lists *spoofer@scam.com*. That is a clear cut example of a spoofed message. 
+
+You should blacklist any address you find in the **REPLY-TO**, **RETURN-PATH**, and **SOURCE IP** field that is not an email address or IP address from which you normally receive mail.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: any address that you find..

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+
+For more information on viewing and understanding email headers, see [View and read Rackspace email headers](/how-to/view-and-read-rackspace-email-headers).
+
+### Combat spoofing
+
+User education is the first line of defense against these types of attacks. If a user receives a spoofed message:
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: If a user receives a spoofed message, they should perform the following tasks:

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+
+ - Blacklist any email address or IP address listed in the **REPLY-TO**, **RETURN-PATH**, or **SOURCE IP** that you have determined to be fraudulent. See [Blacklist addresses, domains, and IP addresses in Rackspace Email](/how-to/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail/) for instructions.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: For instructions, see...

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+
+       **Note:** If you send email from other providers on behalf of your domain, be sure to include their sending servers in the same SPF record entry. Do not create multiple SPF records.
+
+   - **DomainKeys Identified Mail (DKIM)** records assign a digital signature to mail sent from your domain, marking it as authorized mail sent from your domain. If you require instructions to enable DKIM for your Rackspace Cloud Office email, see [Enable DKIM in the Cloud Office Control Panel](/how-to/enable-dkim-in-the-cloud-office-control-panel). Creating a DKIM record is the second step in the process.
+
+   - **Domain Message Authentication Reporting and Compliance (DMARC)** records indicate to recipient mail servers that messages sent from that domain are employing DKIM and SPF sending policies. The recipient mail server then validates the message that you sent by using your DKIM and SPF policies. [Creating a DMARC record policy](/how-to/create-a-dmarc-policy) enables you to enforce DKIM and SPF records.
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: This is the last last in the process. (At end of bullet item)

File: content/rackspace-email/email-spoofing-best-practices.md [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+
+Using record policies protects the integrity of internal emails, as well as the external reputation of your domain. Implementing this protection is a multi-step process that must be carefully followed. For more information, see [Create a DMARC policy](/how-to/create-a-dmarc-policy).
```
Comment (cyrichardson) [inline content/rackspace-email/email-spoofing-best-practices.md]: multi-step process that you must carefully follow.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-87919f7f40/how-to/)

**Pull request merged**